include/alpaka/kernel/TaskKernelGpuUniformCudaHipRt.hpp

```diff
@@ -424,7 +425,7 @@ namespace alpaka
                 // Wait for the kernel execution to finish but do not check error return of this call.
                 // Do not use the alpaka::wait method because it checks the error itself but we want to give a custom
                 // error message.
-                ignore_unused(ALPAKA_API_PREFIX(StreamSynchronize)(queue.m_spQueueImpl->m_UniformCudaHipQueue));
+                ALPAKA_API_PREFIX(StreamSynchronize)(queue.m_spQueueImpl->m_UniformCudaHipQueue);
```

Comment:
Why is this necessary?

Reply:
I think I include dirty data from my branch and by accident removed the fix^^, I am on the way to fix it.